src/Abp/Application/Features/FeatureCheckerExtensions.cs

```diff
@@ -39,67 +39,67 @@ public static string GetValue(this IFeatureChecker featureChecker, int tenantId,
+        /// <returns>True if the current feature's value is "true".</returns>
         public static async Task<bool> IsEnabledAsync(this IFeatureChecker featureChecker, int tenantId, string featureName)
         {
             return string.Equals(await featureChecker.GetValueAsync(tenantId, featureName), "true", StringComparison.OrdinalIgnoreCase);
         }
 
         /// <summary>
-        /// Checks if given feature is enabled.
+        /// Checks if a given feature is enabled.
         /// This should be used for boolean-value features.
         /// 
-        /// This is a shortcut for <see cref="IsEnabled(IFeatureChecker, int, string)"/> that uses <see cref="IAbpSession.TenantId"/> as tenantId.
-        /// So, this method should be used only if TenantId can be obtained from the session.
+        /// This is a shortcut for <see cref="IsEnabled(IFeatureChecker, int, string)"/> that uses <see cref="IAbpSession.TenantId"/>.
+        /// Note: This method should only be used if the TenantId can be obtained from the session.
         /// </summary>
         /// <param name="featureChecker"><see cref="IFeatureChecker"/> instance</param>
         /// <param name="name">Unique feature name</param>
-        /// <returns>True, if current feature's value is "true".</returns>
+        /// <returns>True if the current feature's value is "true".</returns>
         public static bool IsEnabled(this IFeatureChecker featureChecker, string name)
         {
             return AsyncHelper.RunSync(() => featureChecker.IsEnabledAsync(name));
         }
 
         /// <summary>
-        /// Checks if given feature is enabled.
+        /// Checks if a given feature is enabled.
         /// This should be used for boolean-value features.
         /// </summary>
         /// <param name="featureChecker"><see cref="IFeatureChecker"/> instance</param>
         /// <param name="tenantId">Tenant's Id</param>
         /// <param name="featureName">Unique feature name</param>
-        /// <returns>True, if current feature's value is "true".</returns>
+        /// <returns>True if the current feature's value is "true".</returns>
         public static bool IsEnabled(this IFeatureChecker featureChecker, int tenantId, string featureName)
         {
             return AsyncHelper.RunSync(() => featureChecker.IsEnabledAsync(tenantId, featureName));
         }
 
         /// <summary>
-        /// Used to check if one of all given features are enabled.
+        /// Used to check if one or all given features are enabled.
         /// </summary>
         /// <param name="featureChecker"><see cref="IFeatureChecker"/> instance</param>
-        /// <param name="requiresAll">True, to require all given features are enabled. False, to require one or more.</param>
-        /// <param name="featureNames">Name of the features</param>
+        /// <param name="requiresAll">True to require all the given features to be enabled. False to require one or more.</param>
```

Comment:
"True`,` to require `that` all the given features `are` enabled. False`,` to require one or more."

For consistent phrasing with `<summary>`.
There are multiple instances of this.